loopchain/blockchain/blockchain.py

```diff
@@ -523,6 +526,15 @@ def find_preps_by_roothash(self, roothash: Hash32) -> list:
         else:
             return json.loads(preps_dumped)
 
+    @valued_only_lru_cache(maxsize=4, valued_returns_only=True)
+    def roothash_exist_in_db(self, roothash: Hash32) -> Optional[bool]:
+        try:
+            self._blockchain_store.get(BlockChain.PREPS_KEY + roothash)
+        except (KeyError, TypeError):
+            return None
+        else:
+            return True
+
```

Comment:
Did you mean:
```python
    @valued_only_lru_cache(maxsize=4, valued_returns_only=True)
    def roothash_exist_in_db(self, roothash: Hash32) -> Optional[bool]:
        return self.find_preps_by_roothash(roothash)
```

Reply:
If it is close to your original intention, It is better for `roothash_exist_in_db` to follow `find_preps_by_roothash`'s return value.

Reply:
I think 'is_' prefix is ​​not required because it can be identified by type hint. But I will accept your opinion.